tesseract_kinematics/include/tesseract_kinematics/ikfast/ikfast_inv_kin.h

```diff
@@ -122,6 +122,7 @@ class IKFastInvKin : public InverseKinematics
   const std::string& getTipLinkName() const override;
   const std::string& getName() const override;
   const std::string& getSolverName() const override;
+  std::vector<double> getFreeJointValues() const;
```

Comment:
No, I only included it because there was a method to get every other protected class member, but I can remove it

Reply:
There function has been added back now because there is a new method for when using >7 DOF IKfast implementations that allows for the creation of all possible joint combinations. The user may want to extract this data, so I think it makes sense to provide this method now.